demo/kubernetes/rook-client/rook-client.yml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: rook-client
+spec:
+  containers:
+  - name: rook-client
+    image: quay.io/rook/k8s-demo
+    imagePullPolicy: IfNotPresent
+    command: ["sleep", "36500d"]
```

Comment:
Gotcha.  i'll open an issue on this but go with what we have for now since I've got it stable and will not have to do any more manual testing.